source/stylesheets/_core.scss

```diff
@@ -89,6 +89,11 @@ body {
   width: 65%;
 }
 
+.image_size {
```

Comment:
Should we rename this class, `image_size` doesn't really explain the styling.
Perhaps, `mobile_image_resize`?

Reply:
I agree, changing it so its more specific